client-src/js-src/types.d.ts

```diff
@@ -0,0 +1,54 @@
+declare class ChromeStatusHttpError extends Error {
+  resource: string;
+  method: string;
+  status: number;
+  constructor(
+    message: string,
+    resource: string,
+    method: string,
+    status: number
+  );
+}
+
+declare class FeatureNotFoundError extends Error {
+  featureID: number;
+  constructor(featureID: number);
+}
+
+interface FeatureLink {
+  url: string;
+  type: string;
+  information: any;
+  http_error_code: number;
+}
+
+declare interface ChromeStatusClient {
```

Comment:
Now that #3991 is in, I think you can replace this file with `import type {ChromeStatusClient} from '../js-src/cs-client.js';` in the places that use that type.

Reply:
made my life easier. Thank you for doing this